Makefile

```diff
@@ -200,7 +200,7 @@ endif
 
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+	$(KUSTOMIZE) build config/crd | kubectl create -f -
```

Comment:
do you want to fail if the target resource already exists?

Reply:
It's not failing, but we can change that. The `apply` won't work anymore because the CRD is too long. `kubectl` adds an annotation with everything within the `.spec` if we use `apply`. Hence, the change.

Reply:
I see, thanks.